java/src/jmri/TurnoutManager.java

```diff
@@ -264,4 +265,29 @@
 
     public String getDefaultClosedSpeed();
 
+    /**
+     * Get the Interval (in ms) to wait between commands.
+     * Configured in AdapterConfig, stored in memo.
+     *
+     * @param systemName turnout systemName, used to identify the manager's connection in ProxyTurnoutManager
+     * @return the (Turnout) Output Interval in Milliseconds
+     */
+    public int getOutputInterval(@Nonnull String systemName);
+
+    /**
+     * Set the Interval (in ms) to wait between commands.
+     *
+     * @param systemName turnout systemName, used to identify the manager's connection in ProxyTurnoutManager
+     * @param newInterval the new Output Interval in Milliseconds
+     */
+    public void setOutputInterval(@Nonnull String systemName, int newInterval);
+
+    /**
+     * Get end time of latest OutputInterval, calculated from the current time.
+     *
+     * @param systemName turnout systemName, used to identify the manager's connection
+     * @return end time in Milliseconds or null if no interval was set or timer has completed
+     */
+    public LocalTime outputIntervalEnds(@Nonnull String systemName);
```

Comment:
Why does this appear in the public API? It's not a safe thing to access and use, because there's no guarantee it's true by the time you use it.